src/ovirtapi/transform.js

```diff
@@ -707,13 +715,10 @@ const Cluster = {
 //
 //
 const Nic = {
-  toInternal ({ nic }: { nic: ApiNicType }): NicType {
+  toInternal ({ nic, reportedDevice = {} }: { nic: ApiNicType, reportedDevice?: ReportedDevicesType }): NicType {
```

Comment:
Reported devices are part of nic structure and should be parsed inside `toInternal`.